Fix incorrect package location for Log4j2

dhaval24 opened the pull request:
This fixes the incorrect package location for Log4j2 which otherwise could inhibit the users from using this Appender.

The correct location for Log4j AI appender in the code is in the package : 

**com.applicationinsights.log4j.v2**

azurecla commented:
Hi __@xavierdecoster__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jamescon(EXTERNAL)). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

alancameronwills commented:
@xavierdecoster Thanks for this, Xavier. It prompted me to find out from the devs. It turns out that they updated the NuGet very recently and there's now an even better way, using the .config file. So we'll drop this change and I'll update the doc to show that instead.

alancameronwills commented:
David,

Please drop this change and I’ll post something better today.

Thanks
Alan

From: David Wrede [mailto:notifications@github.com]
Sent: 27 October 2015 19:29
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Alan Wills <Alan.Wills@microsoft.com>
Subject: Re: [azure-content] Fix config code for SamplingTelemetryProcessor (#4845)


Assigned #4845<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-content%2fpull%2f4845&data=01%7c01%7cawills%40064d.mgd.microsoft.com%7ca06aba184f5247ae8aac08d2df04e266%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=esB473qhP6NDFY9c2yWmRo0E7e7W5xkA88JsMA1%2bHCE%3d> to @alancameronwills<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2falancameronwills&data=01%7c01%7cawills%40064d.mgd.microsoft.com%7ca06aba184f5247ae8aac08d2df04e266%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=uW2TlhiMF08OVstwwzfqX9z%2fYEJC2EQc%2bd4XL8RSDr0%3d>.

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-content%2fpull%2f4845%23event-447210836&data=01%7c01%7cawills%40064d.mgd.microsoft.com%7ca06aba184f5247ae8aac08d2df04e266%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=WqwJncWWUUfSwMYt5xu%2bTqaEDj6BjAp7Lmps8cUUpSA%3d>.

dhaval24 committed:
Fix incorrect package location for Log4j2 

This fixes the incorrect package location for Log4j2 which otherwise could inhibit the users from using this Appender.
